instructor/function_calls.py

```diff
@@ -118,11 +120,11 @@ def openai_schema(cls):
     @classmethod
     def from_response(
         cls,
-        completion,
-        validation_context=None,
+        completion: T,
```

Comment:
@jxnl Out of curiosity, what's the motivation behind changing this to a generic i.e. where does having `ChatCompletion` fail?